.github/workflows/github-actions-tests.yml

```
@@ -15,27 +15,29 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: [ "3.8", "3.9", "3.10", "3.11", "3.12", "3.13" ]
+        python-version: ["3.9", "3.10", "3.11", "3.12", "3.13", "3.13t" ]
         mongodb-version: [ "4.4", "5.0", "6.0", "7.0", "8.0" ]
```

No spaces around to keep it consistent.
```suggestion
        mongodb-version: ["4.4", "5.0", "6.0", "7.0", "8.0"]
```